app/views/profiles/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= simple_form_for @profile do |f| %>
+  <%= f.input :bio, placeholder: "Tell us about yourself..." %>
+  <%= f.input :batch, placeholder: "1122" %>
+  <%= f.input :show_email %>
+  <%= f.association :languages, collection: Language.all %> <%#  we should use tom select here %>
+  <%= f.button :submit, class: "btn-primary" %>
```

Comment:
We don't have Bootstrap, so this class should be removed.
```suggestion
  <%= f.button :submit %>
```

Reply:
Right forgot we aren't using bootstrap, I have been stuck in bootstrap land in my other project😂